### ISSUE TITLE
Start build notification shows question marks instead of variables in build version

### ISSUE THREAD

**Issue by 47th** [PeteGoo/tcSlackBuildNotifier#39]
We use custom variables to populate build version (vcs version). Start build notification shows question marks instead of variable value.

**Comment by PeteGoo** [comment]
Does this only apply to the "Build Started" notification?

It might be that we have no good way of getting the version information.

**Comment by 47th** [comment]
Yes, this only applies to "Build started" notification. It appears that variable values are being populated at a later stage and therefore appear as question marks. 

I also tried built-in email notifier for the same build and it resolves variables in start build event correctly.

**Comment by PeteGoo** [comment]
Eek. I wonder how they're doing it 
http://javadoc.jetbrains.net/teamcity/openapi/8.1/jetbrains/buildServer/Build.html#getBuildNumber()
